samples/matter/common/src/bridge/bridge_storage_manager.h

```diff
@@ -23,40 +23,32 @@ namespace Nrf
  *		/brd_cnt/ /<uint8_t>/
  *		/brd_ids/ /<uint8_t[brd_cnt]>/
  * 		/brd/
- *			/eid/
- *				/0/	/<uint16_t>/
- *				/1/	/<uint16_t>/
- *				.
- *				.
- *				/n/ /<uint16_t>/
- *			/label/
- *				/0/	/<char[]>/
- *				/1/	/<char[]>/
- *				.
- *				.
- *				/n/ /<char[]>/
- *			/type/
- *				/0/	/<uint16_t>/
- *				/1/	/<uint16_t>/
- *				.
- *				.
- *				/n/ /<uint16_t>/
- *			/bt/
- *				/addr/
- *					/0/	/<bt_addr_le_t>/
- *					/1/	/<bt_addr_le_t>/
- *					.
- *					.
- *					/n/	/<bt_addr_le_t>/
+ *			/0/ /<BridgedDevice>/
+ *			/1/ /<BridgedDevice>/
+ *			.
+ *			.
+ *			/n/ /<BridgedDevice>/
+ *		/ver/ <uint8_t>
  */
 class BridgeStorageManager {
 public:
+	static constexpr auto kMaxUserDataSize = 128;
+
+	struct BridgedDevice {
+		uint16_t mEndpointId;
+		uint16_t mDeviceType;
+		size_t mNodeLabelLength;
+		char mNodeLabel[MatterBridgedDevice::kNodeLabelSize] = { 0 };
+		size_t mUserDataSize = 0;
+		uint8_t *mUserData = nullptr;
```

Comment:
Why only these fields are initialized, but not other ones?

Reply:
Because the other fields are mandatory, so they should be always filled with valid data.